pubspec.yaml

```diff
@@ -24,7 +24,7 @@ dependencies:
   url_launcher: ^5.1.6
   shared_preferences: ^0.5.4
   collection: ^1.14.0
-  flutter_gallery_assets: ^0.1.9
+  flutter_gallery_assets: ^0.2.0
```

Comment:
Wouldn't the old assets still exist in the git history of the repo, so would the repo size actually be reduced?

Reply:
My next step as part of #87 is to rewrite the history to scrap all deleted files